app/analyse/analyser.py

```diff
@@ -48,6 +48,35 @@ def send_mail_for_repo_finish(project_name):
                     username=user.username,
                 )
 
+def get_active_forks(repo, access_token):
+    active_forks = []
+    result_length = 100
+    page = 1
+
+    while result_length == 100:
```

Comment:
might be good to add a limit to how many pages / active forks to search through so it doesn't take too long for large forks, like add page < 15 in the while condition or len(active_forks) <50

Reply:
added a page < 20 limit